test-data/unit/python2eval.test

```diff
@@ -151,15 +151,27 @@ f(**params)
 [out]
 
 [case testFromFutureImportUnicodeLiterals2_python2]
+# coding=utf-8
 from __future__ import unicode_literals
 def f(x): # type: (str) -> None
   pass
 f(b'')
-f(u'')
-f('')
+f(u'щось')
+f('щось інше')
 [out]
-_program.py:5: error: Argument 1 to "f" has incompatible type "unicode"; expected "str"
 _program.py:6: error: Argument 1 to "f" has incompatible type "unicode"; expected "str"
+_program.py:7: error: Argument 1 to "f" has incompatible type "unicode"; expected "str"
+
+[case testUnicodeLiteralsFutureImportAllASCII_python2]
+# coding=utf-8
+from __future__ import unicode_literals
```

Comment:
Could you add tests for some of the other syntactic variations, like `from __future__ import absolute_import, unicode_literals` and `from __future__ import unicode_literals as something_else`?